swfo/brdf_fallback.py

```diff
@@ -400,34 +409,25 @@ def __get_data(dat_filename, param, window):
     return np.nanstd(data, axis=0, ddof=1, keepdims=False)
 
 
-def concatenate_files(infile_paths, outfile, h5_info, doy):
+def concatenate_files(infile_paths, outfile, h5_info, doy, tile_metadata):
     """
     A function to concatenate multiple h5 files
     """
     assert len(infile_paths) == 7
-
-    # TODO create dataset to store uuid for brdf_fallback provenance (use h5_info 
-    # for to id files used in  threshold generation)
-    # h5_info is a dict with key, val pairs: eg : key = '2002.01.01', 
-    # value = 'absolute path to a h5 file'. 
-
-    # if we want to store medata data for averages produced for each day of year
-    # then use following average_metadata dict, which contains the absolute path 
-    # of the files used in average brdf generation. 
-
-    # average_metadata = {key: h5_info[key] for key in h5_info if folder_doy(key) == doy}
-
-    # TODO write dataset called METADATA in outfile (final h5 files containting the 
-    # average brdf datasets for all seven MODIS bands. 
-
+    
+    # set the uuids used in processing average for given day of year
+    tile_metadata['lineage']['doy_average'] = [munge_metadata(h5_info[key]) 
```

Comment:
When/if these names change we may as well fix my lazy implementation and define these as constants.
- referring to doy_average, and brdf_threshold